packages/firestore/src/local/local_store.ts

```diff
@@ -917,15 +915,15 @@ export class LocalStore {
           return PersistencePromise.forEach(removed, (key: DocumentKey) =>
             this.persistence.referenceDelegate.removeReference(txn, key)
           ).next(() => {
-            this.persistence.referenceDelegate.removeTarget(txn, targetData);
+            this.persistence.referenceDelegate.removeTarget(txn, targetData!);
```

Comment:
Can you check if this is still needed? With TypeScript 3.7, `assert` should now be type-aware.

Reply:
Leads to compilation error.